src/main/app.js

```diff
@@ -252,30 +283,15 @@ fluid.defaults("gpii.app", {
                 }
             }
         },
-
-        tray: {
-            type: "gpii.app.tray",
-            createOnEvent: "onPrerequisitesReady",
-            options: {
-                model: {
-                    keyedInUserToken: "{gpii.app}.model.keyedInUserToken",
-                    pendingChanges: "{settingsBroker}.model.pendingChanges"
-                }
-            },
-            // needed as the psp window is used by the tray
-            priority: "after:psp"
-        },
-        waitDialog: {
-            type: "gpii.app.waitDialog",
+        pspReady: {
+            type: "fluid.component",
```

Comment:
I believe a comment here, reminding this must be last in order, would improve even further the chance for errors